lib/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -563,19 +559,11 @@ public synchronized void onTransportUnavailable(ITransport transport, TransportP
 	}
 
 	private class ConnectParams extends TransportParams {
-		private boolean fallback;
-		ConnectParams(ClientOptions options, boolean fallback) {
+		ConnectParams(ClientOptions options) {
 			this.options = options;
-			this.fallback = fallback;
 			this.connectionKey = connection.key;
 			this.connectionSerial = String.valueOf(connection.serial);
-			String[] fallbackHosts;
-			if(fallback && (fallbackHosts = Defaults.getFallbackHosts(options)).length > 0) {
-				fallbackHosts = Defaults.getFallbackHosts(options);
-				this.host = fallbackHosts[random.nextInt(fallbackHosts.length)];
-			} else {
-				this.host = Defaults.getHost(options, host, true);
-			}
```

Comment:
Am I missing something here, as far as I can tell we have removed the fallback host functionality fro Realtime connections in this commit?  The [Random class](https://github.com/gokhanbarisaker/ably-java/blob/8b59f05590e123462f2603ffccdd9a696f4e0434/lib/src/main/java/io/ably/lib/transport/ConnectionManager.java#L825) is not even used anymore, and the connection implementation still references fallback hosts yet does not use them, see https://github.com/gokhanbarisaker/ably-java/blob/8b59f05590e123462f2603ffccdd9a696f4e0434/lib/src/main/java/io/ably/lib/transport/ConnectionManager.java#L574-L578